src/Microsoft.Health.Fhir.R5.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Api.Features.Resources.Bundle
+{
+    /// <summary>
+    /// FHIR R4 specific portions of the bundle handler.
```

Comment:
R5?
If these are the same, I wonder if we can share them somehow

Reply:
R4 and R5 are the same, but STU3's is different